index.d.ts

```diff
@@ -24,6 +24,7 @@ export {
   HTTPError as errors
 }
 
+export const METHODS: (typeof constants.method)[HTTPMethod]
```

Comment:
```suggestion
export const METHODS: HTTPMethod[]
```

That should match the expected type of an array of HTTP method names.